### ISSUE TITLE
Loader does not load a font with the .fnt extension

### ISSUE THREAD

**Issue by dbuzinin** [pixijs/pixijs#8645]
Hello! After upgrading PIXI from version 6.2.2 to 6.5.3, the loader stopped loading fonts with .fnt extensions

In version 6.2.2 this code works:

`const uploadFonts = async (): Promise<void> => {
  return new Promise((resolve) => {
    if (PIXI.BitmapFont.available['Roboto Medium']) {
      resolve();
    } else {
      const loader = new PIXI.Loader();
      loader.add('Roboto', '/fonts/Roboto.fnt').load(() => {
        console.log('FONT resolve');
        resolve();
      });
    }
  });
};`

then
`const text = new PIXI.BitmapText(textStr, { fontName: 'Roboto Medium', fontSize: 18 });
    text.x = 48;
    text.y = 12;`

### Expected Behavior
After loading, the font should be available in PIXI.BitmapFont.available

### Current Behavior
Font not available

### Possible Solution
-
### Environment

- **`pixi.js` version**: 6.5.3
- **Browser & Version**: Chrome 105.0.5195.126
- **OS & Version**: Windows 10

**Comment by bigtimebuddy** [comment]
This sounds like an upgrade issue. Please follow these instructions: https://github.com/pixijs/pixijs/wiki/Upgrading-PixiJS

To confirm it's an upgrade issue, this could happen if you have multiple versions of `@pixi/text-bitmap` in your **package-lock.json**.

**Comment by dbuzinin** [comment]
> This sounds like an upgrade issue. Please follow these instructions: https://github.com/pixijs/pixijs/wiki/Upgrading-PixiJS
> 
> To confirm it's an upgrade issue, this could happen if you have multiple versions of `@pixi/text-bitmap` in your **package-lock.json**.

I followed this instruction, but it doesn't help.

The yarn cache, yarn.lock and the node-modules folder have been removed. Also, I am using pixi-viewport plugin, it uses pixi.js@6.5.1, I tried to fix pixi on the same version. But got the same error.

I tried to comment out the sections of code that use BitmapText and got the following error:

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'start')
    at BatchSystem.setObjectRenderer (core.mjs?4a5d:4148:1)
    at Graphics._renderBatched (graphics.js?ff64:2654:1)
    at Graphics._render (graphics.js?ff64:2608:1)
    at Container.render (display.mjs?2ac7:2160:1)
    at Container.render (display.js?277b:2166:1)
    at Renderer.render (core.mjs?4a5d:9360:1)
    at AbstractRenderer.generateTexture (core.mjs?4a5d:8901:1)
    at Renderer.generateTexture (core.mjs?4a5d:9377:1)
    at ElementView.renderRect (ElementView.ts?02d1:417:1)
    at ElementView.render (ElementView.ts?02d1:107:1)
```

**Comment by bigtimebuddy** [comment]
Yarn has been problematic for us resolving dependencies, especially when you have a plugin like pixi-viewport. Please checkout #6530 for a tip on resolving packages with Webpack, or try to use `npm` which doesn't have this issue.

If this is still a problem, please create a minimal reproduction (preferably a repo on GitHub).

**Comment by oliverbestmann** [comment]
Also. Might be unrelated, but the text format loader seems to be broken. There are multiple "in" loops which should have been "of" loops:
https://github.com/pixijs/pixijs/blob/dev/packages/text-bitmap/src/formats/TextFormat.ts#L82

**Comment by dbuzinin** [comment]
I was able to solve the error problem after upgrading to version 6.5.3. The project uses webpack and yarn. For some reason webpack was not properly resolving dependencies and was importing some packages from esm and some from cjs (like @pixi/extensions).

All problems were solved by adding for webpack the property   `mainFields: ['browser', 'main', 'module'],` https://webpack.js.org/configuration/resolve/#resolvemainfields
